specification/recoveryservices/resource-manager/readme.md

```diff
@@ -28,21 +28,7 @@ These are the global settings for the RecoveryServices API.
 title: RecoveryServicesClient
 description: Recovery Services Client
 openapi-type: arm
-tag: package-2016-12
-```
-
-
-### Tag: package-2016-12
-
-These settings apply only when `--tag=package-2016-12` is specified on the command line.
-
-``` yaml $(tag) == 'package-2016-12'
-input-file:
-- Microsoft.RecoveryServices/stable/2016-12-01/backup.json

```

Comment:
Looks like this file was the only unique thing about the `2016-12` package, so it makes sense that if the file gets deleted, this whole tag does.

Is there a plan to reintroduce this swagger? Why was it ever included?

Reply:
Talking with a couple of my colleagues offline, we agree that unless there is something confidential about the contents of this file it should stay included.

Reply:
Hi @marstr ,

Thanks for the update. The APIs present in Backup.json is now moved to RecoveryServicesBackup spec file, hence was not needed here.

Reply:
I see, makes sense.